addons/filesystem/functions/fnc_chdir.sqf

```diff
@@ -0,0 +1,90 @@
+/**
+ * Change directory.
+ *
+ * Arguments:
+ * 1: Pointer <[STRING]>
+ * 2: Filesystem <HASHMAP>
+ * 3: Raw path to target directory <STRING>
+ * 4: Creates a directory if it is not found <BOOL> (Optional)
```

Comment:
In which cases, chdir is called with _create = false ?

Reply:
AE3_filesystem_fnc_createFile
AE3_filesystem_fnc_createFolder

later (not implemeneted):
mkdir armaos command